src/middlewares/validate-boolean-params.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+import { errorBuilder } from 'errors/errors';
+import { openapi } from 'utils/load-openapi';
+
+// get all boolean query parameters from openapi
+const booleanParams = [];
+_.forEach(openapi.paths, (endpoints) => {
+  _.forEach(endpoints, (endpoint) => {
+    booleanParams.push(..._(endpoint.parameters)
+      .filter({ in: 'query', schema: { type: 'boolean' } })
+      .map('name')
+      .value());
+  });
+});
```

Comment:
I believe this logic can be simplified by `[_.reduce](https://lodash.com/docs/#reduce)`.

Reply:
Maybe I'm using it wrong but it seems a lot more complicated. Thoughts?

Reply:
You're right, I tried it and doesn't improve a lot. Feel free to ignore my comment here.